src/data/showcases.js

```diff
@@ -1211,6 +1211,15 @@ export const Showcases = [
     source: null,
     tags: ["daotool", "governance"],
   },
+  {
+    title: "Begin Wallet",
+    description:
+      "Begin Wallet is a light Cardano Wallet, with one of the best user experiences.",
```

Comment:
```suggestion
      "Begin Wallet is a light Cardano Wallet, with ... [these features] ... .",
```
The original wording is too unverifiable and subjective to include.  What about the CIP-0013 specialised URL support which is relatively unique for Begin Wallet?

Reply:
There is no guideline on how comprehensive the source needs to be but of course the criticism is justified. (That said we had other wallets who first only where front end open source cases, and at some point followed suit.)

Reply:
I'm OK approving this if you or @francisluz can provide some objective language for the `description:` 😎

Reply:
**p.p.s.** specifically Begin Wallet was verified by @MadOrkestra to support Payment URIs and other [unspecified] CIP-0013 protocols: https://github.com/cardano-foundation/CIPs/pull/882/files

Reply:
Hey guys, thanks a lot for the warm welcome and the feedback. I'll do a better description highlighting the features. Thanks for the reminder on CIP13. I'll put it in a more regular user-friendly way without mentioning CIPs, okay?

@katomm and @rphair, regarding the Open source part, is it possible to extend the tags to have a partial open source category? That's the category we've been around other repos on Cardano, which reflects the reality of Begin.

Reply:
Looks good about the feature inventory.  Mostly resolved with https://github.com/cardano-foundation/developer-portal/pull/1381/commits/d8597de590766f06a24738c41761ad28295cb31f & will continue with 1 more code suggestion in next review.